tests/services/librespeed.bats

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bats
+load ../test-helper
+
+@test "$clinom services librespeed info" {
+  run "../${clicmd}" services librespeed info
```

Comment:
Need to fix the directories.

Remove `../` in `run`.

Reply:
my bad. fixed